### PR TITLE
[LLT-4357] natlab: Add debug logs to windowsVM

### PR DESCRIPTION
**Pull request by lcruz99**
### Problem
There's something fishy going on with Windows VM as it's randomly failing tests with no obvious reason.

### Solution
Add logs so we can debug.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] changelog.md is updated

### CODE REVIEW

**Comment by jjanowsk** [comment]
As I understand this is supposed to be temporary?

File: nat-lab/tests/utils/connection_util.py [hunk 1]
```diff
@@ -104,10 +104,18 @@ class ConnectionTag(Enum):
...
+        ConnectionTag.DOCKER_OPEN_INTERNET_CLIENT_1
+    ),
+    ConnectionTag.DOCKER_OPEN_INTERNET_CLIENT_2: (
+        ConnectionTag.DOCKER_OPEN_INTERNET_CLIENT_2
+    ),
+    ConnectionTag.DOCKER_OPEN_INTERNET_CLIENT_DUAL_STACK: (
+        ConnectionTag.DOCKER_OPEN_INTERNET_CLIENT_DUAL_STACK
+    ),
+    ConnectionTag.DOCKER_INTERNAL_SYMMETRIC_CLIENT: (
+        ConnectionTag.DOCKER_INTERNAL_SYMMETRIC_GW
```
Comment (stalowyjez) [inline nat-lab/tests/utils/connection_util.py:117]: Ooops

File: nat-lab/tests/utils/vm/windows_vm_util.py [hunk 2]
```diff
@@ -39,6 +39,22 @@ async def new_connection() -> AsyncIterator[Connection]:
...
+
+        connection.create_process(["route", "print"]).run(
+            stdout_callback=on_stdout,
+            stderr_callback=on_stdout,
+        )
+        connection.create_process(["ipconfig/all"]).run(
+            stdout_callback=on_stdout,
+            stderr_callback=on_stdout,
+        )
+
```
Comment (stalowyjez) [inline nat-lab/tests/utils/vm/windows_vm_util.py:57]: It looks like some debug, do you really want to merge it to master?

Comment (lcruz99) [reply nat-lab/tests/utils/vm/windows_vm_util.py:57]: Yes, this is a temporary merge

**Comment by lcruz99** [comment]
> As I understand this is supposed to be temporary?

Yes, it's an attempt to caught a bug in Windows VM that happens approximately once every night (~1/25 runs), in the debug session we could not reproduce it, so we decided to add this temporarily to `main`.

**Review by jjanowsk — APPROVED** [review]

**Pull request merged**